app/screens/backup/BackupRoot.js

```diff
@@ -0,0 +1,193 @@
+// @flow
+import { shell } from 'electron';
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import type { RouterHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { WrapperWith2SideBars, Button, Link } from '/basicComponents';
+import { fileEncryptionService } from '/infra/fileEncryptionService';
+import { fileSystemService } from '/infra/fileSystemService';
+import { localStorageService } from '/infra/storageService';
+import { bottomLeftCorner, bottomRightCorner, topLeftCorner, topRightCorner, smallHorizontalSideBar } from '/assets/images';
+import type { WalletMeta } from '/types';
+import smColors from '/vars/colors';
+
+const Wrapper = styled.div`
```

Comment:
make this component shorter, too many empty space

Reply:
Please clarify. I was going for the same height as, for example, the overview page. You do mean the height of the component?

Reply:
there is a lot of empty space, the component is very high and it looks bad

Reply:
ok